playwright.config.js

```diff
@@ -50,11 +50,14 @@ export default defineConfig({
   /* Fail the build on CI if you accidentally left test.only in the source code. */
   forbidOnly: !!process.env.CI,
 
+  /* Limit the number of failures */
+  maxFailures: 1,
```

Comment:
Let’s fail as quickly as possible so as not to waste CI time. I’ve gone back and forth on setting an exception for local runs, but I found it more productive and helpful in a local environment as well.

Reply:
Potentially, we can increase the `maxFailures` again and increase retries. I’d like to have the E2E tests fail as early as possible for now to learn which tests exactly are our flaky tests and first try to make them less flaky.